pkg/api/types.go

```diff
@@ -2138,3 +2138,8 @@ type RangeAllocation struct {
 	// a single allocated address (the fifth bit on CIDR 10.0.0.0/8 is 10.0.0.4).
 	Data []byte `json:"data"`
 }
+
+const (
+	// "default-scheduler" is the name of default scheduler.
+	DefaultSchedulerName = "default-scheduler"
```

Comment:
@smarterclayton was saying we should make these constants versioned, so you should make the same change to pkg/api/v1/types.go